README.md

```diff
@@ -32,94 +62,18 @@ Verifies the following:
 * Vertices are used in at least one edge (except metadata)
 * [WIP] Edges exist only between defined vertices
 
-# Graph
```

Comment:
Ugh you can't comment on lines that weren't changed in GitHub...

I know you didn't change this, but nit: line 60 should be "Edges'" or "Edge"

Reply:
Doesn't that mean line 59 should be "vertex" and not "vertices"?

Reply:
Assuming so and closing this! Let me know if I am wrong. English, right?